src/main/java/ca/team2706/frc/robot/RobotState.java

```diff
@@ -0,0 +1,10 @@
+package ca.team2706.frc.robot;
+
+public enum RobotState {
+
+    ROBOT_INIT,
+    DISABLED,
+    AUTONOMOUS,
+    TELEOP,
+    TEST

```

Comment:
Maybe we could also add a shutdown hook here as well, but I can add that in a later pull request